docs/veriwell/getting-started.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 0
+title: Getting Started
+description: About multi-language verification tool
+keywords: [Multi chain contract, smart contracts]
+---
+
+### Developer
+
+Developers need to know the contract's address or deployment hash. Additionally, they must be aware of the OS and compiler version used during deployment. Depending on the programming language of the contract, the compilation process may or may not depend on the OS. For example, Arbitrum's Stylus generates different bytecode depending on the OS used during compilation. Similarly, compiler versions can affect compatibility; certain functions may be backward-compatible with earlier versions, while others may not.
+
+As a result, even if the OS and compiler version differ between deployment and verification, successful verification might still be possible in some cases. However, in most scenarios, the verification process fails without a matching environment. Therefore, developers need to know the OS and compiler version used at the time of deployment to replicate the same build and compilation environment, ensuring successful verification.
+
+Once the contract address or hash and the deployment environment are configured, developers can upload the code for compilation. The code is compiled according to the specified environment, and if the compiled output matches the on-chain code, the contract is verified successfully.
+
+### User
+
+No preparation is required. Simply access the VeriWell website and input the address of the contract you wish to verify. If the contract is verified, you will be redirected to a results page where you can review the contract's code along with explanations provided by AI support about the code's functionality.
```

Comment:
`input the the address of the contract and the source code zip file`?

Reply:
finish editing sentence